packages/wp-story-editor/src/index.js

```diff
@@ -35,38 +38,42 @@ import { initializeTracking } from '@web-stories-wp/tracking';
 /**
  * Internal dependencies
  */
+import * as apiConfig from './api';
 import './style.css'; // This way the general editor styles are loaded before all the component styles.
 
 /**
  * Initializes the web stories editor.
  *
  * @param {string} id       ID of the root element to render the screen in.
- * @param {Object} config   Story editor settings.
+ * @param {Object} settings Story editor settings.
  * @param {Object} flags    The flags for the application.
  */
-const initialize = (id, config, flags) => {
+const initialize = (id, settings, flags) => {
   const appElement = document.getElementById(id);
 
   // see http://reactcommunity.org/react-modal/accessibility/
   setAppElement(appElement);
 
-  updateSettings(config.locale);
+  updateSettings(settings.locale);
 
   initializeTracking('Editor');
 
   render(
     <FlagsProvider features={flags}>
       <StrictMode>
-        <App config={config} />
+        <StoryEditor settings={settings} config={apiConfig}>
+          <PostLock />
+          <StatusCheck />
+        </StoryEditor>
       </StrictMode>
     </FlagsProvider>,
     appElement
   );
 };
 
 const initializeWithConfig = () => {
-  const { id, config, flags } = window.webStoriesEditorSettings;
-  initialize(id, config, flags);
+  const { id, config: settings, flags } = window.webStoriesEditorSettings;
+  initialize(id, settings, flags);
```

Comment:
Dealing with both `config` and `settings` is quite confusing, especially when doing things like `config: settings` or `<Context.Provider value={{ ...settings, config }}>`, so everything is merged anyway.

Can we just use either `settings` or `config` across the board, not both?

Reply:
Yes, I agree, this was just an attempt to separate callbacks and settings from `webStoriesEditorSettings`. Probably I should just go with `config`.